recipe/meta.yaml

```diff
@@ -224,7 +255,9 @@ outputs:
         - python
         - numpy
         - pip
-        - setuptools
+        # see https://github.com/pytorch/pytorch/issues/136541
+        - setuptools <=72.1.0  # [win]
+        - setuptools  # [not win]
         - pyyaml
         - requests
         - six
```

Comment:
Does pytorch still depend on `six`? 🤔 

(also, more openmp mentions below; if we end up enabling that on windows)

Reply:
Can we keep a general cleanup somewhat separate from this PR? :)

Reply:
IMO anything suspicious/broken we come across (which doesn't take major surgery) should be done as soon as we're doing new pushes here. Deleting obsolete dependencies seems to fit the bill, but I won't insist. 🤷